dev-client/src/components/menus/MenuList.tsx

```diff
@@ -16,15 +16,14 @@
  */
 
 import {Children} from 'react';
+import {StyleSheet, View} from 'react-native';
 import {Divider} from 'react-native-paper';
 
-import {Column} from 'terraso-mobile-client/components/NativeBaseAdapters';
-
 export type ButtonListProps = React.PropsWithChildren;
 
-export function ButtonList({children}: ButtonListProps) {
+export function MenuList({children}: ButtonListProps) {
   return (
-    <Column mt="12px" mb="24px" space="6px">
+    <View style={styles.root} accessibilityRole="menu">
       <Divider />
       {Children.map(children, child => {
```

Comment:
1. First I was wondering: Do you not need a `key` for the Divider and maybe also children?

2. Then I googled and haven't read thru this, but seems like React docs discourage use of the `Children` API: https://react.dev/reference/react/Children#alternatives

Reply:
Interesting - will take a look.

Reply:
After reading the above, I've removed the use of `Children` in the list component.